test/test_numerizer_en.rb

```diff
@@ -36,17 +36,19 @@ def test_straight_parsing
       500 => '5 hundred',
       999 => 'nine hundred and ninety nine',
       1_000 => 'one thousand',
-      1_200 => ['twelve hundred', 'one thousand two hundred'],
+      1_200 => ['twelve hundred', 'one thousand two hundred', 'thousand two hundred'],
+      1100 => ['thousand hundred', 'thousand and hundred'],
       17_000 => 'seventeen thousand',
       21_473 => 'twentyone-thousand-four-hundred-and-seventy-three',
       74_002 => 'seventy four thousand and two',
       99_999 => 'ninety nine thousand nine hundred ninety nine',
       100_000 => '100 thousand',
       250_000 => 'two hundred fifty thousand',
       1_000_000 => 'one million',
+      1_001_100 => ['one million one thousand and one hundred', 'million thousand hundred'],
```

Comment:
is it possible to not have the last case work for all of these, I'm not sure that you would actually want that to parse.